benchmarks/benchmark_serving/README.md

```diff
@@ -0,0 +1,16 @@
+# Benchmarking Text Generation
```

Comment:
We use `kebab-case` in all of the other dirs in this project. Prefer to keep with that convention and rename this to `benchmarks/chat-py/` and change the other to be `benchmarks/chat-k6/`.

Reply:
done